doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -74,8 +75,21 @@ lose their role as parameters. For example:
 
    Notation "'IF' c1 'then' c2 'else' c3" := (c1 /\ c2 \/ ~ c1 /\ c3) (at level 200, right associativity).
 
-Symbols that start with a single quote with 3 or more characters must be single quoted.
-For example, the symbol `'ab` is represented by `''ab'` in the notation string.
+Symbols that start with a single quote and followed by at least 2
+characters must be single quoted.  For example, the symbol `'ab` is
+represented by `''ab'` in the notation string. Quoted strings can be used in
+notations: they must begin and end with two double quotes.
+Embedded spaces in these strings are
+part of the string and do not contribute to the separation
+between notation tokens. To embed double quotes in these strings, use four
+double quotes (e.g. the notation :g:`"A ""I'm an """"infix"""" string symbol"" B"`
+defines an infix notation whose infix symbol is the string
+:g:`"I'm an ""infix"" string symbol"`). Symbols may contain
+double quotes without being strings themselves (as e.g. in symbol :g:`|"|`) but notations with such symbols can be
+used only for printing (see :ref:`Use of notations for printing <UseOfNotationsForPrinting>`).
+In this case, no spaces are allowed in the symbol.  Also, if the
+symbol starts with a double quote, it should be surrounded with single
```

Comment:
```suggestion
symbol starts with a double quote, it must be surrounded with single
```
?  Or else describe when it's permitted, or be clear that the confusion will only occur on the part of the users, etc.

Reply:
Adopted